styles.css

```diff
@@ -117,11 +117,299 @@
   background-repeat: no-repeat;
 }
 
+.tonic {
+  width: 295px;
+  height: 320px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 auto;
+  display: block;
+}
+
+.header-tonic {
+  width: 295px;
+  height: 56px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 35px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 1rem;
+}
+
+.canopy {
+  width: 55px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.back {
+  width: 88px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.description {
+  list-style: none;
+  height: 37px;
+}
+
+ul.description {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 5%;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.more-info {
+  width: 295px;
+  height: 106px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-left: 1rem;
+}
+
+.languages {
+  width: 29px;
+  height: 52px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  margin-left: 1rem;
+  list-style: none;
+}
+
+.lang {
+  margin: 10px;
+  padding: 7px;
+  border-radius: 7px;
+  margin-top: 2px;
+  background-color: #ebebff;
+  width: 55px;
+  height: 33px;
+  border: 0;
+}
+
+#lan-1 {
+  width: 47px;
+  height: 31px;
+  color: #6070ff;
+  font-size: 15px;
+}
+
+#lan-2 {
+  width: 43px;
+  height: 31px;
+  color: #6070ff;
+  font-size: 15px;
+}
+
+#lan-3 {
+  width: 84px;
+  height: 32px;
+  color: #6070ff;
+  font-size: 15px;
+}
+
+.see {
+  font-family: 'Poppins', sans-serif;
+  width: 120px;
+  height: 84px;
+  color: #396df2;
+  border-radius: 8px;
+  margin-left: 1rem;
+  background: #fff;
+  border: 1px solid blue;
+  font-size: 19px;
+}
+
+#see-1 {
+  width: 120px;
+  height: 50px;
+}
+
+.stories {
+  width: 295px;
+  height: 88px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 35px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 1rem;
+}
+
+.works {
+  margin-top: 30px;
+  height: 100vh;
+  margin-bottom: 30px;
+}
+
+.works-1 {
+  margin-top: 50px;
+  height: 100vh;
+  margin-bottom: 30px;
+}
+
+.myself {
+  margin-top: 95px;
+  height: 100vh;
+  margin-bottom: 30px;
+}
```

Comment:
- [ ] Kindly align the about myself section with flexbox as stated in the project requirements. Currently, they are not properly aligned compared to the Figma design. Kindly fix this as well :wink:
- Your Design
![image](https://user-images.githubusercontent.com/102699958/219921469-b68dc376-e306-47ab-a15c-112b23caf62d.png)
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/219921654-2a06bbdd-0f59-40ae-a50d-ecb9365d2656.png)

- Please try to stick to the Figma Design as much as possible.